### DISCUSSION TITLE
External hard drive (serving data)

### DISCUSSION THREAD

**Discussion by vipinkashyap** [serverpod/serverpod#2688] [Q&A] [answered]
Hi all,
I have an external hard disk with a ton of photos . I'd like to have the serverpod server access these pictures and then give me access to these via endpoints . I'd like this to be an experiment that I'm only running locally so any help or ideas are appreciated.

**Comment by vipinkashyap** [comment]
> Hi all,
> I have an external hard disk with a ton of photos . I'd like to have the serverpod server access these pictures and then give me access to these via endpoints . I'd like this to be an experiment that I'm only running locally so any help or ideas are appreciated.

Ive seen some articles about mounting the hard drive to a docker container but i don't have any idea about the Docker world . Maybe that can be an option ?

**Reply by Isakdl** [reply, marked as answer]
Yes this is a good direction if you are running the server inside a docker container. Take a look at the docker docs here: https://docs.docker.com/engine/storage/volumes/ for how to mount volumes.

In essence, you map a folder on your disc into a folder inside the running container.

If you don't care or need docker you can just run the server directly on your machine too and then you should have direct access to the file system.

**Reply by vipinkashyap** [reply]
Thank you for getting back I'm going to try what you've recommended :)
